docs/Release_notes.md

```diff
@@ -46,10 +46,15 @@ Create a New ‘Trajectory Follower Wrapper’ Control Plugin: Prototypes a new
 - Pull Requests: [carma-platform PR #2447](https://github.com/usdot-fhwa-stol/carma-platform/pull/2447), [carma-platform PR #2448](https://github.com/usdot-fhwa-stol/carma-platform/pull/2448), [carma-platform PR #2430](https://github.com/usdot-fhwa-stol/carma-platform/pull/2430), [carma-platform PR #2379](https://github.com/usdot-fhwa-stol/carma-platform/pull/2379), [carma-platform PR #2419](https://github.com/usdot-fhwa-stol/carma-platform/pull/2419)
   
 **Fixes**  
-- Pull Requests: [carma-platform PR #2333](https://github.com/usdot-fhwa-stol/carma-platform/pull/2333), [carma-platform PR #2387](https://github.com/usdot-fhwa-stol/carma-platform/pull/2387), [carma-platform PR #2458](https://github.com/usdot-fhwa-stol/carma-platform/pull/2458), [carma-platform PR #2531](https://github.com/usdot-fhwa-stol/carma-platform/pull/2531), [carma-platform PR #2504](https://github.com/usdot-fhwa-stol/carma-platform/pull/2504), [carma-platform PR #2535](https://github.com/usdot-fhwa-stol/carma-platform/pull/2535)
```

Comment:
Not a comment on this specific line, but these release notes are missing some content from our final version. For example, the second paragraph describing the changes in CARMA Platform ("With the upgrade to ROS 2 Humble in this release ...") is not included here. Could you please do a full check of the release notes here to make sure everything from our final version is included?